app/core/components/FormFields/SelectField.tsx

```diff
@@ -35,6 +37,12 @@ const SelectField: React.FC<SelectFieldProps> = ({
     setSelectedValue(event.target.value);
   };
 
+  useEffect(() => {
+    if (value !== undefined) {
+      setSelectedValue(value);
+    }
+  },[value, setSelectedValue])
+
```

Comment:
is this needed?

I think this line further down does more or less the same:

```
          value={selectedValue || ""}
```

Oh! Wait. I see, you added a value prop that we receive now. But I don't see where you are using that new prop.

The thing is that before what we were doing was that forms using this component are usually [ValidatedForms](https://www.remix-validated-form.io/reference/validated-form). Which can receive a defaultValues prop that has an object with properties and values. This line further up gets the value for this component from that defaultValues property by it's name:

```
  const [selectedValue, setSelectedValue] = useControlField<string>(name);
```